tests/bluetooth/tester/src/btp_ias.c

```diff
@@ -0,0 +1,62 @@
+/* btp_ias.c - Bluetooth IAS Server Tester */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/bluetooth/services/ias.h>
+
+#include "btp/btp.h"
+#include "zephyr/sys/byteorder.h"
+
+#include <zephyr/logging/log.h>
+#define LOG_MODULE_NAME bttester_ias
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#define CONTROLLER_INDEX 0
```

Comment:
What is the purpose of defining the `CONTROLLER_INDEX` in each source file, which seemingly is also always the same value? Couldn't we just use a single definition?

Reply:
as number of services supported by tester grow  plan to do some refactoring in tester app in upcoming months, but for this PR I'd prefer it follows how other services are implemented

Reply:
Can we resolve this then?

Reply:
Should we perhaps create an Enhancement issue for it, so we can track it?

Reply:
I've created and issues for this (assigned to myself)
https://github.com/zephyrproject-rtos/zephyr/issues/53814